src/hotspot/share/runtime/vm_version.cpp

```diff
@@ -30,7 +30,7 @@
 
 void VM_Version_init() {
   VM_Version::initialize();
-
+  guarantee(VM_Version::supports_cx8(), "Support for 64-bit atomic operations in required in this release");
```

Comment:
Typo: "in required in". Also, no need to mention "this release" at all?
Suggestion for message: "JVM requires platform support for 64-bit atomic operations"

Reply:
Or the simpler change:
s/in required/is required/

Reply:
Message tweaked: Support for 64-bit atomic operations is required